CMakeLists.txt

```diff
@@ -62,10 +62,12 @@ endif()
 
 # gsrunner
 if(ENABLE_GSRUNNER)
-	if (NOT WIN32)
-		message(WARNING "GSRunner is only supported on Windows and may not build on your system")
+	if (NOT WIN32 and NOT APPLE)
```

Comment:
`and` needs to be `AND` otherwise
```
CMake Error at CMakeLists.txt:65 (if):
  if given arguments:

    "NOT" "WIN32" "and" "NOT" "APPLE"

  Unknown arguments specified
  ```